oneandone/provider.go

```diff
@@ -40,6 +40,7 @@ func Provider() terraform.ResourceProvider {
 			"oneandone_ssh_key":           resourceOneandOneSshKey(),
 			"oneandone_block_storage":     resourceOneandOneBlockStorage(),
 			"oneandone_image":             resourceOneandOneImage(),
+			"oneandone_baremetal":         resourceOneandOneBaremetalServer(),
```

Comment:
we're missing some documentation for this resource (in `./website/r/baremetal.html.markdown` and `./website/oneandone.erb`) - but this otherwise LGTM 👍